pkgs/applications/audio/pianoteq/default.nix

```
@@ -0,0 +1,146 @@
+{ pkgs ? (import <nixpkgs> {}) }:
+with pkgs;
```

```suggestion
{ ... }:
```

please specify the packages explicitly.

---

Done in e520318a0050cb0c8ecee9b04009f38082a9b0c6 , thanks